schematic/visualization/tangled_tree.py

```diff
@@ -258,11 +258,16 @@ def _get_ca_alias(self, conditional_requirements: list[str]) -> dict[str, str]:
                 value: attribute
         """
         ca_alias: dict[str, str] = {}
+        extracted_conditional_requirements = []
```

Comment:
Thank you so much for the change @BryanFauble ! I am curious if this part should be wrapped in a function? If this part gets wrapped in a function, we could test it with different conditional requirements string and make sure they are all formatted correctly.

Reply:
I tested it with some conditional requirements from HTAN data model, and they all worked!

Reply:
I added unit test for `_get_ca_alias` function: https://github.com/Sage-Bionetworks/schematic/pull/1480
Maybe that will help too?

Reply:
Thanks - I'll merge this in